src/views/PurchaseOrder.vue

```diff
@@ -126,14 +126,17 @@ export default defineComponent({
           shopifyProductUPC: [],
           arrivalDate: [],
           quantityOrdered: [],
-          facilityId: []
+          facilityId: [],
```

Comment:
I think we should set the `facilityId` and `externalFacilityId` as is in save method and rather manipulate in apply method only

Reply:
Updated the code.